packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -2478,6 +2488,10 @@ class WebpackCLI implements IWebpackCLI {
           this.logger.raw(printedStats);
         }
       }
+
+      if (options.failOnWarnings && (stats.hasErrors() || stats.hasWarnings())) {
+        process.exit(1);
```

Comment:
```suggestion
        process.exitCode = 1;
```
I think we should terminate gracefully

Reply:
@vankop I have a doubt, If we not exit here, process will not terminate on warning/errors from webpack in watch mode (with option named as `--fail-on-warnings`). Will this be accurate behavior?

Reply:
Okay great, will update this then 👍🏽